### DISCUSSION TITLE
VS Code ou Visual Studio Community

### DISCUSSION THREAD

**Discussion by developsouza** [balta-io/2801#89] [Perguntas e Respostas] [answered]
Gostaria de saber se tem alguma restrição em acompanhar o curso usando o Visual Studio Community, ou, se vou ter algum problema para acompanhar sem ser pelo VS Code, e, caso eu acompanhe no VS Code, conseguirei me orientar quando for para o VS Community? Grato.

**Comment by andrebaltieri** [comment, marked as answer]
Não tem não... porém recomendo começar com VS Code pois te ajuda menos.... fazendo mais coisas manuais você fixa mais...
👉 https://www.youtube.com/watch?v=8f0CKHcf8vc

**Reply by developsouza** [reply]
Show, não tinha pensado nessa perspectiva, obrigado, seus ensinamentos e didática são incríveis. **Deus o abençoe**

**Reply by cmsouzaac** [reply]
Realmente um ótimo professor, estou gostando muito da didática e da plataforma. @andrebaltieri acabei de assisti o video de CLI.

**Reply by andrebaltieri** [reply]
<3
